spec/feature/feature_spec.rb

```diff
@@ -0,0 +1,60 @@
+require_relative '../../lib/take_away.rb'
+
+describe 'User Stories' do
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+ it 'See a list of dishes with prices' do
+   takeaway = Takeaway.new("fabios","+555222","7 james street")
+   takeaway.create_menu
+   expect(takeaway.menus[0].menu_items.map.each(&:item))
+     .to eq(["CHICKEN CACCIATORE", "VEGGIE BURGER", "TURKEY MILANESE",
+        "GENNARO’S FAMOUS PORCHETTA", "ITALIAN STEAK FRITES",
+        "LAMB CHOPS SCOTTADITTO", "HARISSA-SPICED AUBERGINE"])
+ end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+
+it 'Select some number of several available dishes ' do
+  menu = Menu.new
+  order = Order.new("jim","7","north road","nw10","+555444")
+  order.add_to_cart(menu,menu.itemised[0],2)
+  order.add_to_cart(menu,menu.itemised[0],1)
+  expect(order)
+    .to have_attributes(cart:[[["CHICKEN CACCIATORE", "£13.95"], 2],
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)